src/hotspot/cpu/arm/c1_MacroAssembler_arm.cpp

```diff
@@ -53,7 +53,7 @@ void C1_MacroAssembler::inline_cache_check(Register receiver, Register iCache) {
   bind(verified);
 }
 
-void C1_MacroAssembler::build_frame(int frame_size_in_bytes, int bang_size_in_bytes) {
+void C1_MacroAssembler::build_frame(int frame_size_in_bytes, int bang_size_in_bytes, int max_monitors) {
```

Comment:
So the `max_monitors` param is added, but not use of it.
Is someone else doing the 32-bit ARM port?

Reply:
Hopefully :-) I currently can't do it, though. With fast-locking (and the rest of Lilliput) behind an experimental flag, this is probably ok for now?